.github/workflows/qa.yaml

```diff
@@ -43,7 +43,7 @@ jobs:
 
   tests:
     name: Tests
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-24.04
```

Comment:
A small, nice TODO to revert to -latest once GH points 24.04 to it? :)

Reply:
What now that we are using variables? :joy: